grai-integrations/source-mysql/src/grai_source_mysql/models.py

```diff
@@ -0,0 +1,146 @@
+from enum import Enum
+from typing import Any, Dict, List, Optional, Union
+
+from pydantic import BaseModel, Field, root_validator, validator
+
+
+class mysqlNode(BaseModel):
```

Comment:
Small thing but standard python uses `PascalCase` for classes rather than `camelCase`.